### PR TITLE
[Fix] Outline fixes

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
Fliplet/fliplet-studio#6420

## Description
We need important to override some bootstrap styling and remove the outline.

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: scss/core/base.scss [hunk 1]
```diff
@@ -356,22 +356,25 @@ table.mce-item-table {
   box-shadow: none;
```
Comment (tonytlwu) [inline scss/core/base.scss:356]: If the class is used on something that has a custom box-shadow, then box-shadow shouldn't be removed.

```suggestion
```

File: scss/core/base.scss [hunk 2]
```diff
@@ -356,22 +356,25 @@ table.mce-item-table {
...
+    outline: none;
   }
   
   @include above($tabletBreakpoint) {
     box-shadow: none;
 
     &:not(:hover) {
-      box-shadow: 0px 0px 0px $focusOutlineWidthTablet $focusOutlineColorTablet;
+      box-shadow: 0px 0px 0px $focusOutlineWidthTablet $focusOutlineColorTablet !important;
+      outline: none;
```
Comment (tonytlwu) [inline scss/core/base.scss:368]: This is the same static value for all breakpoints. Why can't we just have this declared once for mobile, which will them apply to all breakpoints?

```suggestion
```

**Comment by MaksymShokin** [comment]
@tonytlwu 
Fixed. Seems to work fine

File: scss/core/base.scss (hunk 2 above)
Comment (MaksymShokin) [reply scss/core/base.scss]: Fixed

File: scss/core/base.scss (hunk 1 above)
Comment (MaksymShokin) [reply scss/core/base.scss]: Fixed

**Pull request merged**